projects/fsastorefrontlib/src/cms-components/fnol/fnol-navigation/fnol-navigation.component.ts

```diff
@@ -83,18 +84,13 @@ export class FNOLNavigationComponent implements OnInit, OnDestroy {
       ])
         .pipe(
           map(([submittedFormData, uploadedContent, userRequest]) => {
-            let claimCopy = { ...claimData };
+            const claimCopy = _.cloneDeep(claimData);
             if (submittedFormData && submittedFormData.content) {
-              let contentDataCopy = {
-                ...claimData.configurationSteps[this.activeStepIndex]
-                  .stepContent.contentData,
```

Comment:
This object has another one inside which would also need to be copied in order for this to work. This looks very ugly and confusing, so `lodash` was introduced to avoid that.

Reply:
Turns out, for this we don't need lodash. JSON.stringify works well because our objects, though deeply nested, contain simple strings and object literals, without any symbols or functions, which would require usage of lodash.